evalml/automl/automl_search.py

```diff
@@ -454,27 +459,41 @@ def __init__(
         sampler_method="auto",
         sampler_balanced_ratio=0.25,
         allow_long_running_models=False,
-        _ensembling_split_size=0.2,
         _pipelines_per_batch=5,
         automl_algorithm="default",
         engine="sequential",
         verbose=False,
         timing=False,
+        _holdout_set_size=0.1,
```

Comment:
does it make sense to make the default behavior the current behavior? (setting to 0)

Reply:
Yea, I was kind of getting at this to some degree in my comment about refactoring some of the nested conditional logic.  I think doing this will probably necessitate looking into how `split_data()` handles accepting a 0.  I think Chris wants it to return a None in the case that the split is empty.

Reply:
Ended up bumping `holdout_set_size` to public and will keep as default/